src/Input/Command.php

```diff
@@ -382,4 +385,20 @@ protected function io(): Interactor
     {
         return $this->_app ? $this->_app->io() : new Interactor;
     }
+
+    /**
+     * Get ProgressBar instance.
+     */
+    protected function progress(int $total = null): ProgressBar
+    {
+        if ($this->_progressBar === null) {
+            $this->_progressBar = new ProgressBar(null, $this->writer());
```

Comment:
is same progressbar reusable again n again without resetting?

Reply:
oh no... i was too focused on minimal resource usage but this way we'll have the same instance of the progressbar every time. thanks very much.

Nevertheless I do not see why someone would venture to call this method several times in the same command

Reply:
yea that may not be too normal but not impossible either. maybe in rare cases someone would like to have multiple phases and each phase with different steps that would show progresses. 
we can revisit on this later though.